### ISSUE TITLE
Error creating a Packed MSIX

### ISSUE THREAD

**Issue by geoffhop** [marcinotorowski/MSIX-Hero#114]
Hi there
Thanks for your great product.
I have a previously created MSIX file (originally created from an MSI Install by the Microsoft MSIX Packaging Tool) which is working fine for installing.
I used MSIX Hero 2.1.0.0 to unpack that MSIX package to a folder on disk without issue.
When attempt to pack that unpacked folder (not changing the folder or it's contents in any way as I just want to test that it can be repacked) I get an error from MSIX Hero stating
"Root element is missing"

I have attached the a screenshot for reference

![2021-04-23_12-41-49](https://user-images.githubusercontent.com/51848170/115819449-6af92e00-a431-11eb-8c74-07c0f144dc19.png)

Any ideas on how to overcome this error?

Thanks in advance.
Geoff

**Comment by marcinotorowski** [comment]
Hi, this seems to be the same as the one from #108.
Would it be possible to create a small repro package, or at to send me a manifest file (AppxManifest.xml) from the root?

**Comment by geoffhop** [comment]
Hi Marcin
Thanks for the replyI have attached the AppxManifest file.I am not in a position to create a small repro package until I am back at work on Tuesday Australian Western Standard Time.
Thanks in advanceGeoff 
    On Saturday, 24 April 2021, 12:04:35 am AWST, Marcin Otorowski ***@***.***> wrote:  
 
 


Hi, this seems to be the same as the one from #108.
Would it be possible to create a small repro package, or at to send me a manifest file (AppxManifest.xml) from the root?

**Comment by marcinotorowski** [comment]
Hi,

there seems to be no attachment. Could you send me the file to hello (at) msixhero.net?

**Comment by geoffhop** [comment]
Hi MarcinPlease find the file attached.
RegardsGeoff
    On Sunday, 25 April 2021, 04:43:44 pm AWST, Marcin Otorowski ***@***.***> wrote:  
 
 


Hi,

there seems to be no attachment. Could you send me the file to hello (at) msixhero.net?

**Comment by marcinotorowski** [comment]
I was unable to reproduce the reported error having only the manifest file. However, I noticed the manifest has no root declaration for UTF-8, so I changed a few bits in the packer to better handle this case (commit a02b4ac9)

There is now a preview build available (2.1.4.0) with this fix and some other improvements and changes:
https://github.com/marcinotorowski/MSIX-Hero/releases/tag/v2.1.4.0-pre

The MSIX can be downloaded from this page. Let me know if this works for you.
